### ISSUE TITLE
Make a portable build

### ISSUE THREAD

**Issue by a8underscore** [crimist/trakx#35]
Currently with the install.sh it installs it in backround and stuff, it would be great if we can get a binary that we can run anywhere and close it any time without haveing to worry about "installing" trakx.

btw great project

**Comment by crimist** [comment]
The reason I have an "install" script is because it currently reads the index page and the config from files meaning they need to be present on the system. I'll think of a good way to embed the html page and default settings for the tracker and than you'll be able make a portable version of it.

> btw great project

Thanks :)

**Comment by a8underscore** [comment]
An idea I have is you can have config file and index file in same directory as binary

**Comment by crimist** [comment]
Sure, I'll add that and also include the default configs within the executable. That way you can run it with no files present.

**Comment by a8underscore** [comment]
Awesome

**Comment by crimist** [comment]
I've included the changes in 776cd27ac81ef5ea3ca5edf89d7997d80525c7f5.

Now it's as easy as:
```sh
git pull https://github.com/crimist/trakx; cd trakx
go install .
./trakx run
```
